src/DssSpell.t.sol

```diff
@@ -283,12 +283,13 @@ contract DssSpellTest is DssSpellTestBase {
     }
 
     function testNewOrUpdatedChainlogValues() private { // make private to disable
-        _testNewOrUpdatedChainlogValues([
-            "MCD_PSM_GUSD_A_JAR",
-            "MCD_PSM_GUSD_A_INPUT_CONDUIT_JAR",
-            "MCD_PSM_PAX_A_JAR",
-            "MCD_PSM_PAX_A_INPUT_CONDUIT_JAR"
-        ]);
+        bytes32[] memory keys = new bytes32[](4);
+        keys[0] = "MCD_PSM_GUSD_A_JAR";
+        keys[1] = "MCD_PSM_GUSD_A_INPUT_CONDUIT_JAR";
+        keys[2] = "MCD_PSM_PAX_A_JAR";
+        keys[3] = "MCD_PSM_PAX_A_INPUT_CONDUIT_JAR";
+
+        _testNewOrUpdatedChainlogValues(keys);
```

Comment:
i think is much cleaner now, with the risk of declaring a same index twice (`[1] = 'A'; [1] = 'B';`), the test should fail if being inputted an empty space in the array, so having to declare the length of the array (at L286) should be contrasted to the total amount of new or updated chainlog values 👏